src/components/CreatePlaylistModal/CreatePlaylistModal.jsx

```diff
@@ -53,6 +54,20 @@ const CreatePlaylistModal = () => {
         }
     }
 
+    const checkVideoInWatchLater = () => {
+        const isExist = watchLater.find(item => item._id === video._id);
+        return isExist ? true : false;
```

Comment:
`const checkVideoInWatchLater = () =>  watchLater.find(item => item._id === video._id) ? true : false`

Reply:
> `const checkVideoInWatchLater = () => watchLater.some(item => item._id === video._id)`

Yes, this one is more cleaner, thanks for sharing @mohammed-sas